ggml-cuda.h

```diff
@@ -18,6 +18,8 @@ extern "C" {
 #define GGML_CUDA_MAX_DEVICES       16
 
 GGML_API void   ggml_init_cublas(void);
+GGML_API bool   ggml_cublas_loaded(void);
```

Comment:
Add comment that we differentiate between "initialized" and "loaded" state. The former means we have called `ggml_init_cublas()` but it's not guaranteed that there has been a CUDA device available, in which case `ggml_cublas_loaded()` is false.

Reply:
done